impr(quotes): add English quotes (connorcarey)

connorcarey opened the pull request:
### Description

<!-- Please describe the change(s) made in your PR -->

### Checks

- [X] Adding quotes?
  - [X] Make sure to include translations for the quotes in the description (or another comment) so we can verify their content.
- [ ] Adding a language or a theme?
  - [ ] If is a language, did you edit `_list.json`, `_groups.json` and add `languages.json`?
  - [ ] If is a theme, did you add the theme.css?
    - Also please add a screenshot of the theme, it would be extra awesome if you do so!
- [x] Check if any open issues are related to this PR; if so, be sure to tag them below.
- [x] Make sure the PR title follows the Conventional Commits standard. (https://www.conventionalcommits.org for more info)
- [x] Make sure to include your GitHub username inside parentheses at the end of the PR title

<!-- label(optional scope): pull request title (your_github_username) -->

<!-- I know I know they seem boring but please do them, they help us and you will find out it also helps you.-->

Closes #

<!-- the issue(s) your PR resolves if any (delete if that is not the case) -->
<!-- please also reference any issues and or PRs related to your pull request -->
<!-- Also remove it if you are not following any issues. -->

<!-- pro tip: you can mention an issue, PR, or discussion on GitHub by referencing its hash number e.g: [#1234](https://github.com/monkeytypegame/monkeytype/pull/1234) -->

<!-- pro tip: you can press . (dot or period) in the code tab of any GitHub repo to get access to GitHub's VS Code web editor Enjoy! :) -->

FFUV reviewed (changes requested):
Gurren Lagann fan I see, looks good to me besides num 7063  not sure if there is grammatical checking for quotes but it uses "bro" a lot.

connorcarey commented:
> Gurren Lagann fan I see, looks good to me besides num 7063 not sure if there is grammatical checking for quotes but it uses "bro" a lot.

Thanks for bringing this to my attention you're completely right. I removed that quote and fixed a double exclamation mark in another while checking the other two.

Miodec reviewed (approved):
Thanks
